dashboard/app/controllers/followers_controller.rb

```diff
@@ -25,14 +25,24 @@ def student_register
       return render 'student_user_new', formats: [:html]
     end
 
-    Retryable.retryable on: [Mysql2::Error, ActiveRecord::RecordNotUnique], matching: /Duplicate entry/ do
-      if @user.save && @section&.add_student(@user)
-        sign_in(:user, @user)
-        redirect_to root_path, notice: I18n.t('follower.registered', section_name: @section.name)
-        return
+    if current_user && current_user.display_captcha? && !verify_recaptcha
+      current_user.increment_section_attempts
```

Comment:
from a functional point of view, this is not useful. The user is already over the limit.